tailwind.config.js

```diff
@@ -16,6 +16,18 @@ module.exports = {
 				light: '0px 0px 5px rgba(0, 0, 0, 0.1)',
 				hover: '0px 0px 15px rgba(0, 0, 0, 0.25)',
 			},
+      textColor: {
+				skin: {
+					white: 'var(--color-white)',
+					error: 'var(--color-error)',
+				},
+			},
+			backgroundColor: {
+				skin: {
+					medium: 'var(--color-medium)',
+					error: 'var(--color-error)',
+				},
+			},
```

Comment:
Is het niet overzichtelijker om de kleuren in je tailwind config te definiëren in plaats van nog eens in een css file en dan de variabele te gebruiken?

Reply:
Ahja, jij deed da zo in de extensie dus dacht dat dat common practice was... Zal het aanpassen.